glean_parser/util.py

```diff
@@ -374,21 +374,38 @@ def format_error(
         return f"{filepath}:\n{textwrap.indent(content, '    ')}"
 
 
-def parse_expires(expires: str) -> datetime.date:
+def parse_expiration_date(expires: str) -> datetime.date:
     """
     Parses the expired field date (yyyy-mm-dd) as a date.
     Raises a ValueError in case the string is not properly formatted.
     """
     try:
         return date_fromisoformat(expires)
-    except ValueError:
+    except (TypeError, ValueError):
         raise ValueError(
             f"Invalid expiration date '{expires}'. "
             "Must be of the form yyyy-mm-dd in UTC."
         )
 
 
-def is_expired(expires: str) -> bool:
+def parse_expiration_version(expires: str) -> int:
+    """
+    Parses the expired field version string as an integer.
+    Raises a ValueError in case the string does not contain a valid
+    positive integer.
+    """
+    try:
+        version_number = int(expires)
```

Comment:
Do we want to implicitly convert from a string to an int here, or require that the versions be ints in the input file?  I can see this both ways, I guess, but the alternative would be `if isinstance(expires, int):`

Reply:
Good idea, yes, let me require ints in the file :)